sdk/storage/Azure.Storage.Files.Shares/src/Models/ShareFileRangeInfo.cs

```diff
@@ -37,6 +37,11 @@ public class ShareFileRangeInfo
         /// </summary>
         public IEnumerable<HttpRange> Ranges { get; internal set; }
 
+        /// <summary>
+        /// Clear ranges for the file.
+        /// </summary>
+        public IEnumerable<HttpRange> ClearRanges { get; internal set; }
```

Comment:
is `clear` a verb or adjective in this context?

Reply:
hmm. I can se there's api `ClearRangeAsync`. Maybe we should call it `ClearedRanges` ?

Reply:
This was pattern-matched from Page Blob - https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Blobs/src/Models/PageRangesInfo.cs#L43